AmazonChimeSDK/AmazonChimeSDKTests/common/CommonTestCase.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CommonTestCase.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@testable import AmazonChimeSDK
+import XCTest
+ 
+class CommonTestCase: XCTestCase {
```

Comment:
I don't see any test cases here? If this is to provide mocks / setup for other tests to use, then perhaps something like `MockProvider`?

Also, why can't the other tests do the setup themselves?

Reply:
Several test cases require the same mock and there is a lot of duplicate code without a place to host common mocks.